postgres/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+name: harmony-postgres
+services:
+  postgres:
+    image: postgres:14
+    restart: always
+    environment:
+      POSTGRES_USER: ${POSTGRES_USER:-postgres}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
+      POSTGRES_DB: ${POSTGRES_DB:-postgres}
+    ports:
+      - "5432:5432"
```

Comment:
@tahitihat can you tell me a bit more about the port conundrum. Are we running everything on the same server? So do we need to make sure that the harmony postgres doesn't interfere with the druid postgres?

Reply:
Per the use case you're talking about, yes, everything will run on one machine so port conflicts will need to be prevented. Ideally any code we add to fit configure on one machine does not constrain Harmony users to a single-machine setup since that is far from optimal

Reply:
Generally the default port can be used, but still good practice to leave the port configurable. I recall doing this on my initial PR to avoid local conflicts.